cmd/explaintest/r/explain_easy.result

```diff
@@ -76,9 +76,9 @@ TopN_7	1.00	root	test.t2.c2:asc, offset:0, count:1
     └─TableScan_13	10000.00	cop	table:t2, range:[-inf,+inf], keep order:false, stats:pseudo
 explain select * from t1 where c1 > 1 and c2 = 1 and c3 < 1;
 id	count	task	operator info
-IndexLookUp_11	1.11	root	
+IndexLookUp_11	11.08	root	
```

Comment:
https://github.com/pingcap/tidb/pull/10984/files#diff-20ac3c94a9605bc56ae644dfd45569f8R234 changes the row count estimation. Maybe we need to first support the pk in index's histogram.